docs/api-reference.md

```diff
@@ -11,16 +11,17 @@ route: '/reference/api'
 - [`act`](/reference/api#act)
 - [`cleanup`](/reference/api#cleanup)
 - [`addCleanup`](/reference/api#addcleanup)
+- [`removeCleanup`](/reference/api#removecleanup)
 
 ---
 
 ## `renderHook`
 
-```js
-function renderHook(
-  callback: function(props?: any): any,
+```ts
+renderHook: (
+  callback: (props?: any): any,
   options?: RenderHookOptions
-): RenderHookResult
+) => RenderHookResult
```

Comment:
I'm not sure these read better or easier than the old ones, especially for non-TS users.

Reply:
> non-TS users

I think that's fair, I didn't consider that.